packaging/nnfw.spec

```diff
@@ -157,7 +151,7 @@ If you want to get coverage info, you should install runtime package which is bu
 %endif # coverage_build
 
 %define build_options -DCMAKE_BUILD_TYPE=%{build_type} -DTARGET_ARCH=%{target_arch} -DTARGET_OS=tizen \\\
-        -DEXTERNALS_BUILD_THREAD=%{nproc} -DBUILD_MINIMAL_SAMPLE=ON -DNNFW_OVERLAY_DIR=$(pwd)/%{overlay_path} \\\
+        -DEXTERNALS_BUILD_THREAD=%{nproc} -DNNFW_OVERLAY_DIR=$(pwd)/%{overlay_path} \\\
```

Comment:
`BUILD_MINIMAL_SAMPLE` flag is modified from "ON" to "OFF". Is it your intention?

Reply:
It's a mistake. I reverted this change.